cli/Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -204,10 +207,14 @@ public struct Environment: Environmenting {
 
     public var cacheDirectory: AbsolutePath {
         let baseCacheDirectory: AbsolutePath
-        if let cacheDirectoryPathString = variables["XDG_CACHE_HOME"],
+        if let cacheDirectoryPathString = variables["TUIST_XDG_CACHE_HOME"],
            let cacheDirectory = try? AbsolutePath(validating: cacheDirectoryPathString)
         {
             baseCacheDirectory = cacheDirectory
+        } else if let cacheDirectoryPathString = variables["XDG_CACHE_HOME"],
+                  let cacheDirectory = try? AbsolutePath(validating: cacheDirectoryPathString)
+        {
+            baseCacheDirectory = cacheDirectory
         } else {
```

Comment:
```suggestion
        if let cacheDirectoryPathString = variables["TUIST_XDG_CACHE_HOME"] ?? variables["XDG_CACHE_HOME"],
           let cacheDirectory = try? AbsolutePath(validating: cacheDirectoryPathString)
        {
            baseCacheDirectory = cacheDirectory
        } else {
```

can skip some repetition like this. I would suggest doing that in the rest of this file, too